packages/malloy-vscode/src/extension/webviews/help_page/App.tsx

```diff
@@ -11,67 +11,51 @@
  * GNU General Public License for more details.
  */
 
-import React, { useState } from "react";
+import React, { useEffect, useState } from "react";
 import styled from "styled-components";
+import Markdown from "markdown-to-jsx";
```

Comment:
In other parts of the project (docs and composer), we’re using `remark` and `unified` for markdown parsing/rendering. In those cases we need direct control over how the markdown renders, but it may still be worth looking into whether we can use those instead of adding another markdown parser to our dependencies (`markdown-to-jsx`).

Reply:
Removed sidebar code.